enterprise/frontend/test/metabase-enterprise/formatting/whitelabel.cy.spec.js

```diff
@@ -218,44 +229,44 @@ describeWithToken("formatting > whitelabel", () => {
       });
     });
 
-    it("should reflect logo change on admin's dashboard", () => {
-      signInAsAdmin();
+    it("changes should reflect on admin's dashboard", () => {
       cy.visit("/");
       checkLogo();
     });
 
-    it("should reflect logo change while signed out", () => {
+    it("changes should reflect while signed out", () => {
+      signOut();
       cy.visit("/");
       checkLogo();
     });
 
-    it("should reflect logo change on user's dashboard", () => {
+    it("changes should reflect on user's dashboard", () => {
```

Comment:
For some reason if I put these tests inside of one (according to the best practices), total running time is **longer** by 5s on average o.O

Maybe because there is a log of signing out and in. In any way, I left them separated.